### ISSUE TITLE
Suggestion: Another Signal lights (EU used)

### ISSUE THREAD

**Issue by Cronosus** [Railcraft/Railcraft#732]
http://verejna-doprava.eu/navestidla/seradovaci.htm
http://verejna-doprava.eu/navestidla/

These types of signals are used on train stations ( for marshalling )
is possible to make it?

thanks in advance :)

**Comment by LizzyTrickster** [comment]
I suggested in the IRC chat about some extra signals and CJ's response was something along the lines of *it would be too much of a pain to add the needed signals to the API index*.

**Comment by Cronosus** [comment]
yes, but theres already dual signal lights, then i think its only about colors (maybe gui to change colors only, or another signel heads... gui is better way)

**Comment by CovertJaguar** [comment]
I'm not sure I really understand what is being requested here....a blue aspect?

**Comment by Cronosus** [comment]
aspect will be good, i want to build this:
![navestidlo_odj_5](https://cloud.githubusercontent.com/assets/5349950/14042314/069be3d6-f279-11e5-8373-aeeca93fbccd.gif)

**Comment by CovertJaguar** [comment]
If you want to change the colors and names of existing Aspects, that can be easily done with a resource pack.

**Comment by Cronosus** [comment]
not change; red, yellow and green is required too
just add new aspects, or add new signals with these colors

**Comment by Landrash** [comment]
@Cronosus What would be the use-case of these new signals?

**Comment by Cronosus** [comment]
i already post it (first post), they're used in Europe as signals for speed on track (all colors incl. white),
and on stations for sorting trains on tracks (blue and white)

**Comment by Landrash** [comment]
Ok, let me rephrase that. 
What would these new signals that are **blue** instead of **green/red** do that you cant do with the current signalling system? Is it just a cosmetic change you want or is there a added functionality your requesting?

**Comment by Cronosus** [comment]
i'm not sure what you mean, but i dont want "change" colors which are used now (red,green,yellow), i want add two colors (white,blue).
there is functionality:
- blinking white is calling
- red + white is shifting
- blue is shifting prohibited
- white is shifting allowed

**Comment by Generalcamo** [comment]
This doesn't seem useful in a Railcraft Setting.
